man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
+          Defaults to <literal>root</literal>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelaySec=</varname></term>
+        <listitem>
+          <para>Specifies the network fixed amount of delay to all packets going out of the interface. Defaults to unset.</para>
```

Comment:
"Specifies a fixed amount of delay to add to all packets going out..."

and maybe "Defaults to 0"? The user may wonder what the behavior of "unset" is, while 0 is equivalent (afaik) and clearer.

Reply:
> "Defaults to 0"

No, we usually use "Defaults to unset", it means using kernel's default value.

Reply:
No unset means kernel defaults

Reply:
It doesn't say that "unset" == "kernel's default" however.

Reply:
Yeah, we need to explain what "unset" means.